W/Wayland/build_tarballs.jl

```diff
@@ -44,12 +44,13 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Expat_jll"; compat="2.2.10"),
-    Dependency("Libffi_jll"; compat="~3.2.2"),
+    Dependency("Expat_jll"),
+    Dependency("Libffi_jll"),
```

Comment:
Please don't remove compat bounds, they're there for a reason: #3048.

Reply:
Ah, I missed that those require bounds... The problem is that only the newer versions of the JLLs have been built for aarch64-freebsd, so if the compat bounds are kept then downstream packages won't be able to support aarch64-freebsd (which would be fine with me given what a PITA rebuilding all the packages is). Is it possible to rebuild JLLs at a specific version? If we could rebuild Expat/Libffi at those versions to get aarch64-freebsd builds then I think everything would work.

Reply:
Also, like in all other PRs, _changing_ compat bounds means that you must change the package number.

Reply:
Ok, but my question remains: is it possible to rebuild the JLLs at those compat versions? Because otherwise I don't know how to get Wayland building on aarch64-freebsd.

Reply:
Just revert to the old version, and then revert the revert.

Reply:
Alrighty, with the new JLLs the build succeeds on aarch64-freebsd.